projenrc/codecov.ts

```diff
@@ -24,6 +24,7 @@ export class CodeCovWorkflow extends Component {
     });
 
     this.workflow.addJob('collect', {
+      runsOn: ['ubuntu-latest'],
```

Comment:
@mrgrain we should make this either required in projen or default to ubuntu-latest right

Reply:
Huh. That might have changed tbh.